docs/concepts/cluster-administration/sysctl-cluster.md

```diff
@@ -132,17 +132,19 @@ to schedule those pods onto the right nodes.
 
 The use of sysctl in pods can be controlled via annotations on the PodSecurityPolicy.
 
-Here is an example, it authorizes binding user creating pod with corresponding
-_safe_ and _unsafe_ sysctls.
+Sysctl annotations represents the key of a whitelist of allowed safe and unsafe sysctls
+in a pod spec. It's a comma-separated list of plain sysctl names or sysctl patterns
+(which end in * ). The string * matches all sysctls.
```

Comment:
s/*/`*`/

Reply:
In other words, please, surround asterisk by apostrophes.

Reply:
Per the style guidelines, isn't the use of apostrophes around the asterisk not recommended? 
See https://deploy-preview-7014--kubernetes-io-master-staging.netlify.com/docs/home/contribute/style-guide/#use-normal-style-for-string-and-integer-field-values

Reply:
Maybe you're right but it's not just a string. Also if you look at https://deploy-preview-7666--kubernetes-io-master-staging.netlify.com/docs/concepts/cluster-administration/sysctl-cluster/#podsecuritypolicy-annotations you see that now asterisk doesn't emphasized and looks not so good. In the PSP doc, asterisk is highlighted everywhere.